DataManagementSystem/Agent/RequestOperations/MoveReplica.py

```diff
@@ -247,122 +388,173 @@ def dmTransfer( self, opFile ):
     allReplicasCorrupted = replicas["AllReplicasCorrupted"]
 
     if noReplicas:
-      gMonitor.addMark( "ReplicateFail" )
-      self.log.error( "Unable to replicate", "File %s doesn't exist" % ( lfn ) )
+      self.log.error("Unable to replicate", "File %s doesn't exist" % (lfn))
       opFile.Error = 'No replicas found'
       opFile.Status = 'Failed'
-      return S_ERROR()
     elif missingAllReplicas:
-      gMonitor.addMark( "ReplicateFail" )
-      self.log.error( "Unable to replicate", "%s, all replicas are missing" % ( lfn ) )
+      self.log.error("Unable to replicate", "%s, all replicas are missing" % (lfn))
       opFile.Error = 'Missing all replicas'
       opFile.Status = 'Failed'
-      return S_ERROR()
     elif allReplicasCorrupted:
-      gMonitor.addMark( "ReplicateFail" )
-      self.log.error( "Unable to replicate", "%s, all replicas are corrupted" % ( lfn ) )
+      self.log.error("Unable to replicate", "%s, all replicas are corrupted" % (lfn))
       opFile.Error = 'All replicas corrupted'
       opFile.Status = 'Failed'
-      return S_ERROR()
     elif someReplicasCorrupted:
-      gMonitor.addMark( "ReplicateFail" )
-      self.log.error( "Unable to replicate", "%s, replicas corrupted at %s" % ( lfn, someReplicasCorrupted[lfn] ) )
+      self.log.error("Unable to replicate", "%s, replicas corrupted at %s" % (lfn, someReplicasCorrupted[lfn]))
       opFile.Error = 'At least one replica corrupted'
       opFile.Status = 'Failed'
-      return S_ERROR()
     elif missingReplica:
-      gMonitor.addMark( "ReplicateFail" )
-      self.log.error( "Unable to replicate", "%s, missing replicas at %s" % ( lfn , missingReplica[lfn] ) )
+      self.log.error("Unable to replicate", "%s, missing replicas at %s" % (lfn, missingReplica[lfn]))
       opFile.Error = 'At least one missing replica'
       opFile.Status = 'Failed'
+
+    if opFile.Error:
+      if self.rmsMonitoring:
+        self.rmsMonitoringReporter.addRecord({
+            "timestamp": time.mktime(datetime.datetime.utcnow().timetuple()),
+            "host": socket.getfqdn(),
+            "objectType": "File",
+            "operationType": self.operation.Type,
+            "objectID": opFile.FileID,
+            "parentID": self.operation.OperationID,
+            "status": "FileFailed",
```

Comment:
I think the "status": "ReplicateFailed" and this is true for other places. @chaen 
What do you think? My opinion we should be able do distinguish between different activities.

Reply:
I think we can do this with the help of our plotter where we select first the `Operation Type` and then the `Object Type` i.e. `Operation or File`.

Reply:
For example this plot:
![moveReplicaFileStatus](https://user-images.githubusercontent.com/26873907/63189994-db5b8380-c082-11e9-881d-5bc483bfd51c.png)

This is particular to the `MoveReplica` operation and for object type `File`.

Reply:
No, we decided that the status is `Failed`, full stop. The type of activiy is already reflected in the Data Management accounting